core/lib/types/src/tee_types.rs

```diff
@@ -17,6 +19,17 @@ impl fmt::Display for TeeType {
     }
 }
 
+/// Representation of a locked batch. Used in DAL to fetch details about the locked batch to
```

Comment:
If this belongs to the DAL domain, it makes sense to define this type in DAL. AFAICT, there are some TEE-related types defined there already (like `TeeProofGenerationJobStatus`).

Reply:
Moved. ✅